server/verify_test.go

```diff
@@ -1022,27 +1024,34 @@ func TestVerifyAttestationWithSevSnp(t *testing.T) {
 				Nonce:      nonce,
 				TrustedAKs: []crypto.PublicKey{ak.PublicKey()},
 				TEEOpts: &VerifySnpOpts{
-					Getter:             kdsGetter,
-					ReportData:         nonce64,
-					TrustedRoots:       badSnpRoot,
-					AllowDebugTestOnly: true,
+					Validation: SevSnpDefaultPolicyForTest(altNonce),
+					Verification: &sv.Options{
+						Getter:       kdsGetter,
+						TrustedRoots: badSnpRoot,
+					},
 				},
 			},
 			wantErr: "error verifying VCEK certificate",
 		},
-		{
+	}
+	// Production SEV VMs are not run with debug enabled, so this test must be skipped when the
```

Comment:
This comment makes it sound like the following test will fail in prod since they don't have debug enabled. However, we use SevSnpDefaultPolicy, not SevSnpDefaultPolicyForTest, which actually sets `policy.GuestPolicy.Debug = true`.

Reply:
The PolicyForTest sets the *allowance* for a debug bit. By just using the default policy, we're not allowing the debug bit. Production doesn't set the debug bit, so the report will not fail this check that the debug bit is not allowed.

Reply:
Got it, thanks for clarifying!